.github/workflows/performance.yml

```diff
@@ -103,9 +94,27 @@ jobs:
               run: |
                   ./bin/plugin/cli.js perf $(echo $BRANCHES | tr ',' ' ') --tests-branch $GITHUB_SHA --wp-version "$WP_VERSION"
 
+            - name: Get commit timestamp
+              uses: actions/github-script@98814c53be79b1d30f795b907e553d8679345975 # v6.4.0
+              if: github.event_name == 'push'
+              id: commit-timestamp
+              with:
+                  github-token: ${{secrets.GITHUB_TOKEN}}
+                  script: |
+                      const commit_details = await github.rest.git.getCommit({owner: context.repo.owner, repo: context.repo.repo, commit_sha: context.sha});
+                      return parseInt((new Date( commit_details.data.author.date ).getTime() / 1000).toFixed(0))
+
             - name: Publish performance results
               if: github.event_name == 'push'
               env:
                   COMMITTED_AT: ${{ steps.commit-timestamp.outputs.result }}
```

Comment:
I think there's a mismatch between "COMMITTED_AT" and "commit_details.data.author.data" whereby the author can choose a time for the commit but the actual time of commit is different.

we should also be able to skip the network call from JS and use the commit sha directly with `git`. it looks like we're trying to get the seconds since Jan. 1, 1970?

```yaml
-name: Get commit timestamp
 if: github.event_name == 'push'
 id: commit-timestamp
 script: |
    # Print the commit timestamp in Unix epoch seconds
    git show -s '%ct' $GITHUB_SHA
```

In this case `%at` shows the "author time" while `%ct` shows the "commit time." For common work in this repo the difference is largely _when did the author last finish writing the code_ and _when was the code merged into the project_.

Reply:
Nice catch, PR here https://github.com/WordPress/gutenberg/pull/48673